cloudfront.tf

```diff
@@ -25,7 +25,7 @@ module "cdn" {
 
   default_cache_behavior = merge({
     target_origin_id       = "origin_access_identity" # key in `origin` above
-    viewer_protocol_policy = "redirect-to-https"
+    viewer_protocol_policy = "https-only"
```

Comment:
Does this disable the HTTP endpoint? We should always redirect HTTP --> HTTPS.

Reply:
Yes it will block all http requests. I will close this PR for now

the reason I changed is during terraform plan one of the policy check (advisory) failed for cloudfront saying "should use https-only".

Reply:
I see! I think we should change that policy then. Thanks @franklinpashok